index.html

```diff
@@ -4039,8 +4045,38 @@ <h4><dfn>Fullscreen Window</dfn></h4>
   context</a>.
 </ol>
 </section> <!-- /Fullscreen Window -->
-
 </section> <!-- /Switch To Window -->
+<section>
+  <h3><dfn>Consume user activation of Window</dfn></h3>
+
+  <table class="simple jsoncommand">
+    <tr>
+    <th>HTTP Method</th>
+    <th>URI Template</th>
+    </tr>
+    <tr>
+    <td>POST</td>
+    <td>/session/{<var>session id</var>}/window/consume-user-activation</td>
+    </tr>
+  </table>
+
+  <p>The <a>remote end steps</a> are:
+
+  <ol>
+    <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+    return <a>error</a> with <a>error code</a> <a>no such window</a>.
+
+    <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
+
+    <li><p>Let |context| be <a>current browsing context</a>.
+
+    <li><p>Let |consume| be true if |context| has [=transient activation=] or false otherwise.
```

Comment:
Transient activation is applied to Windows, not browsing contexts.

Reply:
Oops, true.